keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -662,11 +662,7 @@ Because this KEP's core implementation(i.e. `SupplementalGroupsPolicy` handling)
 
 #### GA
 
-- At least one of Container Runtimes which is not based on the classic container, gVisor for example, supports the updated CRI and released
-- Assuming no negative user feedback based on production experience, promote after 2 releases in beta.
-- [conformance tests] are added for `SupplementalGroupsPolicy` and `ContainerStatus.User` APIs
-
-[conformance tests]: https://git.k8s.io/community/contributors/devel/sig-architecture/conformance-tests.md
+- No negative user feedback based on production experience, promote after 2 releases in beta.
```

Comment:
I updated GA criteria in two points:

- I removed "gVisor.." thing because it is low-level container runtime, which is not relevant to the feature. And two major CRI runtimes (containerd, CRI-O) already support this feature. I think it's enough to promote this feature to GA
- I removed conformance test criteria. IIUC, CRI runtime dependent features can't be added to the conformance test. So, I removed it.